File: web3rpc/rpc-specs/code-samples/python/debug/runtimeDebugging/setGCPercent.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

percent = 100

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.set_gc_percent(percent)

print(debug_response)

File: web3rpc/rpc-specs/code-samples/javascript/debug/runtimeDebugging/setGCPercent.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
    const percent = 100;
    
    provider.debug.setGCPercent(percent, {}, (err, data, response) => {
        console.log(data);
    });
}
)()